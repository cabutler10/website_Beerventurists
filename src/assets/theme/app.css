html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
  font-size: 14px;
  width: '100%';
}

a {
  text-decoration: none;
}
