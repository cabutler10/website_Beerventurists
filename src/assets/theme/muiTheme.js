import { createMuiTheme } from '@material-ui/core/styles'
import 'typeface-open-sans'
import 'typeface-permanent-marker'
import './app.css'

const status = {
  white: '#ffffff',
  black: '#040001',
  greyLt: '#F7F5F5',
  greyMed: '#656565',
  grey: '#c5c6c7',
  greyDk: '#404040',
  green: '#00942b',
  yellow: '#fce566',
  red: '#FF0012',
  blue: '#015b99',
  blueLt: '#0172c0',
  blueDk: '#014472',
  orange: '#c56113',
  orangeDk: '#c56113',
  orangeLt: '#ebab2f',
  tan: '#FFFBEF',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.blueLt,
      main: status.blue,
      dark: status.blueDk,
    },
    secondary: {
      light: status.orangeLt,
      main: status.orange,
      dark: status.orangeDk,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    useNextVariants: true,
    fontFamily: "'Open Sans', sans-serif",
    h5: {
      letterSpacing: 3,
    },
    h2: {
      fontFamily: "'Permanent Marker', cursive",
    },
    body1: {
      letterSpacing: 1,
      lineHeight: 1.8,
      //TODO is this an error in MUI? not sure why its default is 16 when MUI default is 14
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1,
      fontSize: 14,
    },
    button: {
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.black,
      },
      colorSecondary: {
        color: status.orangeLt,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
    MuiButton: {
      root: {
        padding: '9px 27px',
        borderRadius: 0,
      },
      contained: {
        boxShadow: 'none',
      },
    },
  },
})

export default theme
