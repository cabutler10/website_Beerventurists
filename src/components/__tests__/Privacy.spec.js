import React from 'react'
import { render } from 'react-testing-library'
import Privacy from '../Privacy'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../assets/theme/muiTheme'

describe('Privacy', () => {
  it('renders correctly', () => {
    const mockProps = {
      t: () => '',
    }
    const component = render(
      <ThemeProvider theme={theme}>
        <Privacy {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
})
