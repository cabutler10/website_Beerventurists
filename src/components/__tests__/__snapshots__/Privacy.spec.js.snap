// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-container-uo3ybq"
        id="privacy"
      >
        <h6
          class="MuiTypography-root-1 MuiTypography-h6-18 Hook-title-28my3u"
        >
          privacy.privacy
        </h6>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title1
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms1_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms1_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms1_3
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title2
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms2_1
        </p>
        <ul
          class="MuiList-root-37 MuiList-padding-38"
        >
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms2_list1
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms2_list2
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms2_list3
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms2_list4
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms2_list5
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms2_list6
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms2_list7
              </span>
            </div>
          </li>
        </ul>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms2_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title3
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms3_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms3_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms3_3
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms3_4
        </p>
        <ul
          class="MuiList-root-37 MuiList-padding-38"
        >
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                - Internet Explorer: https://support.microsoft.com/en-us/help/17442/windows-internet-explorer-delete-manage-cookies 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                - Firefox: https://www.mozilla.org/en-US/privacy/websites/#cookies 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                - Chrome: https://support.google.com/accounts/answer/61416?co=GENIE.Platform%3DDesktop&hl=en 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                - Safari: https://support.apple.com/en-gb/guide/safari/manage-cookies-and-website-data-sfri11471/mac 
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                - Opera: https://help.opera.com/en/latest/web-preferences/#cookies 
              </span>
            </div>
          </li>
        </ul>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms3_5
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title4
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms4_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title5
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms5_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title6
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms6_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms6_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title7
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms7_1
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms7_2
        </p>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms7_3
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title8
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms8_1
        </p>
        <a
          class="Hook-link-15pf6ti"
          href="javascript:gaOptout()"
        >
          privacy.terms8_3
        </a>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms8_4
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title9
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms9_1
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title10
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms10_1
        </p>
        <ul
          class="MuiList-root-37 MuiList-padding-38"
        >
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list1
              </span>
            </div>
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list2
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list3
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list4
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list5
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list6
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list7
              </span>
            </div>
             
          </li>
          <li
            class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
          >
            <div
              class="MuiListItemText-root-53"
            >
              <span
                class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
              >
                privacy.terms10_list8
              </span>
            </div>
             
          </li>
        </ul>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title10_2
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms10_2
        </p>
        <h6
          class="MuiTypography-root-1 MuiTypography-subtitle1-19"
        >
          privacy.title11
        </h6>
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
        >
          privacy.terms11_1
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-container-uo3ybq"
      id="privacy"
    >
      <h6
        class="MuiTypography-root-1 MuiTypography-h6-18 Hook-title-28my3u"
      >
        privacy.privacy
      </h6>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title1
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms1_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms1_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms1_3
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title2
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms2_1
      </p>
      <ul
        class="MuiList-root-37 MuiList-padding-38"
      >
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms2_list1
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms2_list2
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms2_list3
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms2_list4
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms2_list5
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms2_list6
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms2_list7
            </span>
          </div>
        </li>
      </ul>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms2_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title3
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms3_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms3_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms3_3
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms3_4
      </p>
      <ul
        class="MuiList-root-37 MuiList-padding-38"
      >
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              - Internet Explorer: https://support.microsoft.com/en-us/help/17442/windows-internet-explorer-delete-manage-cookies 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              - Firefox: https://www.mozilla.org/en-US/privacy/websites/#cookies 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              - Chrome: https://support.google.com/accounts/answer/61416?co=GENIE.Platform%3DDesktop&hl=en 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              - Safari: https://support.apple.com/en-gb/guide/safari/manage-cookies-and-website-data-sfri11471/mac 
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              - Opera: https://help.opera.com/en/latest/web-preferences/#cookies 
            </span>
          </div>
        </li>
      </ul>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms3_5
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title4
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms4_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title5
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms5_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title6
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms6_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms6_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title7
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms7_1
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms7_2
      </p>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms7_3
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title8
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms8_1
      </p>
      <a
        class="Hook-link-15pf6ti"
        href="javascript:gaOptout()"
      >
        privacy.terms8_3
      </a>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms8_4
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title9
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms9_1
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title10
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms10_1
      </p>
      <ul
        class="MuiList-root-37 MuiList-padding-38"
      >
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list1
            </span>
          </div>
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list2
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list3
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list4
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list5
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list6
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list7
            </span>
          </div>
           
        </li>
        <li
          class="MuiListItem-root-41 MuiListItem-default-44 MuiListItem-gutters-49 Hook-listItem-rsrmds"
        >
          <div
            class="MuiListItemText-root-53"
          >
            <span
              class="MuiTypography-root-1 MuiTypography-subheading-8 MuiListItemText-primary-56 Hook-text-jdjt5f"
            >
              privacy.terms10_list8
            </span>
          </div>
           
        </li>
      </ul>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title10_2
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms10_2
      </p>
      <h6
        class="MuiTypography-root-1 MuiTypography-subtitle1-19"
      >
        privacy.title11
      </h6>
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-jdjt5f"
      >
        privacy.terms11_1
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
